example/computeGLCMFeatures.py

```diff
@@ -6,8 +6,12 @@
                                     "<PixelValueMax> <NeighborhoodRadius>")
     sys.exit(1)
 
-im = itk.imread(sys.argv[1])
-mask = itk.imread(sys.argv[2])
+imReader = itk.ImageFileReader.ISS3.New()
```

Comment:
It is preferred to use:

```
Dimension = 3
PixelType = itk.ctype('signed short')
ImageType = itk.Image[PixelType, Dimension]
imReader = itk.ImageFileReader[ImageType].New()
```
It is longer, but it is more explicit, which is especially important for examples.